src/commands/dev/edge/server/http.rs

```diff
@@ -44,27 +44,47 @@ pub async fn http(
                 let req_method = parts.method.to_string();
                 let now: DateTime<Local> = Local::now();
                 let path = get_path_as_str(&parts.uri);
+                let mut full_host: String = "".to_string();
+                match parts.headers.get("host") {
+                    Some(host_header) => {
+                        let subdomains =
```

Comment:
Sounds good- thanks for catching.

Reply:
After, reviewing this more in-depth and testing it... This solution may have a nasty unintended side-effect.

First, I was wrong, getting the host from the URI may not work all the time. With HTTP/1.1, the Host header is required, the URI ends up only being the path. With HTTP/2 (and maybe 3? I'd have to check), it's in the URI. It can be in the Host header, but it's not required (or recommended), so you'll probably have to use a version check to decide how to get the host.

Second, the host is not always a domain, it can be an IP. So with this code, if a user curls with localhost, it works. But if you curl with `127.0.0.1`, it would ping `127.0.0.example.com` and I'm pretty sure that doesn't work. I'd recommend trying to actually parse the domain as an [`std::net::SocketAddr`](https://doc.rust-lang.org/std/net/enum.SocketAddr.html)

Reply:
Also, a constant exists that should probably be used for getting the host header at `hyper::header::HOST`.